src/applications/vaos/appointment-list/components/ConfirmedAppointmentDetailsPage.jsx

```diff
@@ -0,0 +1,224 @@
+              <span>
+                <b>{header}</b>
+              </span>
+            </div>
+
+            <VAFacilityLocation
+              facility={facility}
+              facilityName={facility?.name}
+              facilityId={parseFakeFHIRId(facilityId)}
+              isV2
+              clinicFriendlyName={appointment.participant[0].actor.display}
+            />
+
+            {showInstructions &&
+              isInPersonVAAppointment && (
+                <div className="vads-u-margin-top--3 vaos-appts__block-label">
+                  <AppointmentInstructions
+                    instructions={appointment.comment}
+                    isHomepageRefresh
+                  />
+                </div>
+              )}
+
+            <div className="vads-u-margin-top--3 vaos-appts__block-label">
+              <i
+                aria-hidden="true"
+                className="far fa-calendar vads-u-margin-right--1"
+              />
+              <AddToCalendar
+                summary={`${header}`}
+                description={`instructionText`}
+                location={location}
+                duration={appointment.minutesDuration}
+                startDateTime={moment.parseZone(appointment.start)}
+              />
+            </div>
+
+            <div className="vads-u-margin-top--2 vaos-appts__block-label">
+              <i
+                aria-hidden="true"
+                className="fas fa-print vads-u-margin-right--1"
+              />
+              <a href="#">Print</a>
+            </div>
+
+            <div className="vads-u-margin-top--2 vaos-appts__block-label">
+              <i
+                aria-hidden="true"
+                className="fas fa-clock vads-u-margin-right--1"
+              />
+              <a href="#">Reschedule</a>
+            </div>
+
+            {showCancelButton && (
+              <div className="vads-u-margin-top--2 vaos-appts__block-label">
+                <i
+                  aria-hidden="true"
+                  className="fas fa-times vads-u-margin-right--1"
+                />
+                <button
+                  onClick={() => cancelAppointment(appointment)}
+                  aria-label={`Cancel appointment on ${formatAppointmentDate(
+                    moment.parseZone(appointment.start),
+                  )}`}
+                  className="vaos-appts__cancel-btn va-button-link vads-u-margin--0 vads-u-flex--0"
+                >
+                  Cancel appointment
+                  <span className="sr-only">
+                    {' '}
+                    on{' '}
+                    {formatAppointmentDate(moment.parseZone(appointment.start))}
+                  </span>
+                </button>
+              </div>
+            )}
+          </>
+        )}
+
+      <div className="vads-u-margin-top--3 vaos-appts__block-label">
+        <Link to="/">
+          <button className="usa-button vads-u-margin-top--2">
```

Comment:
Correct me if I'm wrong here, but I like this will produce a button inside a link. It looks like it should just be a link

Reply:
You are correct. I modified to style the link as a button.